app/client/rest/constants.ts

```diff
@@ -2,12 +2,11 @@
 // See LICENSE.txt for license information.
 
 export const HEADER_AUTH = 'Authorization';
+export const HEADER_ACCEPT_LANGUAGE = 'Accept-Language';
 export const HEADER_BEARER = 'BEARER';
 export const HEADER_REQUESTED_WITH = 'X-Requested-With';
 export const HEADER_USER_AGENT = 'User-Agent';
-export const HEADER_X_CLUSTER_ID = 'X-Cluster-Id';
 export const HEADER_X_CSRF_TOKEN = 'X-CSRF-Token';
-export const HEADER_TOKEN = 'Token';
```

Comment:
We are going to have to keep this one, see my other comment about this and the credentials